graphdatascience/system/system_endpoints.py

```diff
@@ -53,10 +61,7 @@ def is_licensed(self) -> bool:
             isLicensed: bool = self._query_runner.run_query(query, custom_error=False).squeeze()
         except Exception as e:
             # AuraDS does not have `gds.debug.sysInfo`, but is always GDS EE.
-            if (
-                "There is no procedure with the name `gds.debug.sysInfo` "
-                "registered for this database instance." in str(e)
-            ):
+            if re.match(r"There is no procedure with the name `gds.*` registered for this database instance", str(e)):
                 isLicensed = True
             else:
                 raise e
```

Comment:
(Unrelated to this PR) I wonder if we should actually raise better errors here. Since the user won't know or need to know that `sysInfo` is used under the hood

Reply:
in practice it should show that the instance is misconfigured 🤔 
(special case for AuraDS as their its configured like this on purpose)

But maybe its also in an on-prem version prevented by the admin to not run some functions